src/commands/dev-env-import-sql.js

```diff
@@ -34,6 +37,24 @@ export class DevEnvImportSQLCommand {
 		const lando = await bootstrapLando();
 		await validateDependencies( lando, this.slug, silent );
 
+		validateImportFileExtension( this.fileName );
```

Comment:
Should we be validating the file name here too?

Reply:
Doesn't appear to be necessary. I think the filename validation we do in `vip import sql` is a result of S3 filename requirements. We _could_ apply the same validation to keep requirements consistent between environments, though it would be an artificial limitation 🤔